docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,36 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+To export the metrics we will use the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus Operator. You can follow up the steps in [Kube Prometheus](https://github.com/coreos/kube-prometheus#installing) project.
```

Comment:
`kube-prometheus` is relative comprehensive monitoring stack and requires quite some resources to deploy. e.g. they don't fit in my 2-nodes cluster, I have to add one more node to fix everything. So I'd suggest something like the following:
```
1. Install Prometheus and Prometheus Operator.
We recommend using [kube-prometheus](https://github.com/coreos/kube-prometheus#installing) in production if you don't have your own monitoring system.
If you are just experimenting, you can only install Prometheus and Prometheus Operator.
```

Reply:
Done. Closing as solved.